.github/workflows/build.yml

```diff
@@ -10,6 +10,7 @@ env:
   S3_BUCKET: ${{ secrets.S3_BUCKET }}
   PROJECT: enhanced-travel-screening
   REGION: ca-central-1
+  DEV_URL: https://dev.covid-ets.bcgov.freshworks.club/
```

Comment:
IMO this should actually come from secrets/env variables rather than being committed - not because of any serious concerns but as a best practice

The region and project are also already stored this way, so we don't need to hardcode those here either 👍

Reply:
Yeah, I was in doubt about this one, was waiting for this comment! Thanks, fixing.